wadas/ui/select_usb_cameras_dialog.py

```diff
@@ -250,6 +251,10 @@ def accept_and_close(self):
                             self.enumerated_usb_cameras[idx].path,
                         )
                         cameras.append(camera)
+
+                        # If db is enabled, insert camera into db.
+                        if (db:=DataBase.get_instance()) and db.enabled:
```

Comment:
Cache the database at the beginning of the method, and then use it where it's needed.